lib/main.dart

```diff
@@ -9,13 +15,32 @@ void main() {
 class ChatApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return MaterialApp(
-      title: 'Chat App',
-      theme: ThemeData(
-        primarySwatch: Colors.green,
-        visualDensity: VisualDensity.adaptivePlatformDensity,
+    return MultiProvider(
+      providers: [
+        ChangeNotifierProvider<LocaleModel>(
+          create: (_) => LocaleModel(),
+        ),
+        ChangeNotifierProvider<ThemeModel>(
+          create: (_) => ThemeModel(ChatAppThemes.defaultTheme),
+        )
+      ],
+      child: Consumer2<LocaleModel, ThemeModel>(
+        builder: (context, localeModel, themeModel, child) => MaterialApp(
+          title: "ChatApp",
+          locale: Provider.of<LocaleModel>(context).getLocale,
+          supportedLocales: [
+            Locale('en', ''),
```

Comment:
drugi argument jest chyba opcjonalny o ile się nie mylę - powinno go się móc usunąc bez problemu

Reply:
tak

Reply:
no to usuń hehe